src/platform/mod.rs

```diff
@@ -4,6 +4,12 @@ pub mod windows;
 #[cfg(feature = "windows")]
 pub use windows::start_event_loop;
 
+#[cfg(feature = "macos")]
```

Comment:
I don't think we'll need these two lines bc 
```
pub use windows::start_event_loop
```
exposes start_event_loop in the scope of platform. We only want to call platform::start_event_loop and not access platform::macos::some_func
Maybe we will need this in the future but for now, I don't think we need it.

Reply:
I tried removing these lines and the code did not run anymore

Reply:
Interesting, we can leave it then